src/Compilers/Core/Portable/Collections/ImmutableArrayExtensions.cs

```diff
@@ -594,5 +595,30 @@ internal static Location FirstOrNone(this ImmutableArray<Location> items)
         {
             return items.IsEmpty ? Location.None : items[0];
         }
+
+        internal sealed class SequenceEqualComparer<T> : IEqualityComparer<ImmutableArray<T>>
```

Comment:
>SequenceEqualComparer [](start = 30, length = 21)

Why a nested class?

Reply:
It's pretty niche. We haven't needed it for the past 7 years. Why not hide it from completion?

Reply:
In that case, consider making it `private` to `AnalyzerConfigSet`.

---
In reply to: [327737635](https://github.com/dotnet/roslyn/pull/38760#discussion_r327737635) [](ancestors = 327737635)

Reply:
Yeah I think that's a good idea. Will-do.